README.md

```diff
@@ -121,14 +121,15 @@ NPM script.
 
 ### check-node-version
 
-Add a `check-node-version` task to package.json, providing the desired version of Node.js
-and NPM that you wish to enforce. Execute it as part of the `test` NPM script as well.
+Add a `check-node-version` task to package.json, providing the desired version of
+Node.js and NPM that you wish to enforce. Execute it as part of the `standards`
+NPM script as well.
 
```

Comment:
Looking at some of the newer pipelines we've upgraded, `npm run check-node-version` is called from the CI config right after `npm ci`. `check-node-version` is being used to validate that the dependencies were installed with the version of npm that we expected. If it's in `npm run standards`, developers will run `check-node-version` when trying to lint the code. 

Could we update the "Execute it..." part to indicate `npm run check-node-version` should be in the CI definition? Open to other ideas as to how to best validate NPM versions.

Reply:
Updated!